app/src/main/java/org/p2p/wallet/send/model/SearchResult.kt

```diff
@@ -15,8 +18,16 @@ sealed class SearchResult(open val addressState: AddressState) : Parcelable {
     data class AddressOnly constructor(
         override val addressState: AddressState,
         val sourceToken: Token.Active? = null,
-        val date: Date? = null
-    ) : SearchResult(addressState)
+        val date: Date? = null,
+        val balance: Long = EMPTY_BALANCE
+    ) : SearchResult(addressState) {
+        @IgnoredOnParcel
+        val isEmpty = balance == EMPTY_BALANCE
+
+        fun copyWithBalance(balance: Long): AddressOnly {
```

Comment:
why just .copy is not enough?

Reply:
it is not a simple copy it is copy with overriding balance